server/app/services/grid_service_instance_deployer.rb

```diff
@@ -82,4 +84,13 @@ def notify_node(node)
     rpc_client = RpcClient.new(node.node_id, 2)
     rpc_client.request('/service_pods/notify_update', [])
   end
+
+  def ensure_volume_instance(node, instance_number)
+    # TODO Maybe separate this functionality into own VolumeScheduler or such
+    self.grid_service.service_volumes.each do |sv|
+      if sv.volume
+        VolumeInstanceScheduler.new.schedule(node, sv, instance_number)
```

Comment:
This sounds more like `VolumeInstanceDeployer.new(sv, instance_number).deploy(node)`... the scheduling has already happened.